sdk/core/Azure.Core/samples/LongRunningOperations.md

```diff
@@ -38,3 +38,72 @@ Console.WriteLine(operation.HasCompleted);
 // hasn't completed yet.
 Console.WriteLine(operation.HasValue);
 ```
+
+## Accessing results for Pageable Operations
+
+A Pageable Operation is use when the service call returns multiple values in pages after the Long Running Operation completes. The results can be access with the `GetValues()`, `GetValuesAsync()` methods which return `Pageable<T>/AsyncPageable<T>` respectively.
+
+To access the result you can iterate over the `Pageable<T>/AsyncPageable<T>`. for more information see [Consuming Service Methods Returning Pageable/AsyncPageable](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/core/Azure.Core#consuming-service-methods-returning-asyncpageablet).
+
+### Using `Value` property
+The `Value` property will contain the `AsyncPageable<T>` results.
+
+```C#
+// create a client
+var client = new TextAnalyticsClient(new Uri("http://example.com"), new DefaultAzureCredential());
```

Comment:
Ideally, these will be code snippets. I need to change TA to target Core with a project reference so I can actually do it

Reply:
Let's add a project reference for now and flip to a package reference when shipped.

Reply:
Hmm how can I avoid the cycle dependency between TA and Core?
Currently, I have TA targeting Core so we can test that the changes added to Core are working. In theory I can undo this and only take the change once Core ships, but prefer not to

Reply:
I don't think there is a cycle

```
TA -> Core
Core.Tests -> TA -> Core
Core.Tests -> Core
```

Reply:
Changos! Azure.Core.Tests... you are right. it should be ok.
Will update